float32 px
float32 py
float32 vx
float32 vy
float32 radius
float32 pgx
float32 pgy
float32 v_pref
float32 theta
float32 px1
float32 py1
float32 vx1
float32 vy1
float32 radius1
---
float32 x_velocity
float32 y_velocity
float32 angle


